### ISSUE TITLE
Offical Mod Suggestions for ATM2

### ISSUE THREAD

**Issue by whatthedrunk** [AllTheMods/ATM-2#3]
Just leave a comment and and link to project.

**Comment by gmunaro** [comment]
Any "Dimension in a block" type of mod... I think there's an alpha of compact machines, and the randomthings mod has a player dimension that can be accessed through a key.

**Comment by adrwa** [comment]
- Replace TOP with HWYLA, more mods support Waila/Hwyla compared to TOP. https://minecraft.curseforge.com/projects/hwyla

- Aether, this one im not sure about tbh, but would give more exploration stuff https://minecraft.curseforge.com/projects/aether-legacy

- Mouse Tweaks https://minecraft.curseforge.com/projects/mouse-tweaks

- Open Modular Turrets https://minecraft.curseforge.com/projects/openmodularturrets

**Comment by gmunaro** [comment]
Please???  https://mods.curse.com/mc-mods/minecraft/227878-minecraft-minions

**Comment by gmunaro** [comment]
Please? 

https://mods.curse.com/mc-mods/minecraft/227878-minecraft-minions

**Comment by gmunaro** [comment]
Quality of life mods: 


https://mods.curse.com/mc-mods/minecraft/mouse-tweaks

https://mods.curse.com/mc-mods/minecraft/238891-dynamic-surroundings

https://mods.curse.com/mc-mods/minecraft/227441-fullscreen-windowed-borderless-for-minecraft

**Comment by gmunaro** [comment]
Enviroemental controller recipes are being messed up because of animalia.

**Comment by gmunaro** [comment]
This mod is a must! https://mods.curse.com/mc-mods/minecraft/redstone-paste

**Comment by Atskadan** [comment]
https://mods.curse.com/mc-mods/minecraft/233071-crafting-tweaks
Very useful for improving the monotony of manual crafting

https://mods.curse.com/mc-mods/minecraft/252676-clipboard 
I often find myself scatterbrained while playing MC and can't focus on one task properly, this mod is extremely helpful in that regard

**Comment by adrwa** [comment]
Galacticraft 1.11.2 https://micdoodle8.com/mods/galacticraft/downloads

**Comment by Atskadan** [comment]
https://minecraft.curseforge.com/projects/chiseled-me
Would be fun with C&B
https://minecraft.curseforge.com/projects/bullseye?gameCategorySlug=mc-mods&projectID=247395
Balanced and highly polished mod that adds a few essential types of arrows

**Comment by Atskadan** [comment]
https://minecraft.curseforge.com/projects/wearable-backpacks?gameCategorySlug=mc-mods&projectID=257572
In the space until Iron Backpacks is ported to 1.11.2, we *really* need a backpack mod IMO.

**Comment by ghost** [comment]
https://mods.curse.com/mc-mods/minecraft/250898-ore-excavation
Comes in handy to break things like slime islands ect

https://mods.curse.com/mc-mods/minecraft/aroma1997s-dimensional-world#t1:description
Allows people to have a place to mine with quarrys ect

https://mods.curse.com/mc-mods/minecraft/246795-viescraft-airships
Brings something new with Airships

**Comment by Atskadan** [comment]
https://minecraft.curseforge.com/projects/just-enough-harvestcraft
Adds JEI support to Harvestcraft blocks which do not have it

**Comment by Atskadan** [comment]
https://minecraft.curseforge.com/projects/clarity?gameCategorySlug=mc-mods&projectID=252145
Makes water much more clear

**Comment by OrdinatorStouff** [comment]
As an alternative to the unusably buggy deep dark: https://mods.curse.com/mc-mods/minecraft/254629-the-beneath

**Comment by adrwa** [comment]
Makes mistakes look good https://minecraft.curseforge.com/projects/missingtextures

**Comment by rosareven** [comment]
## Already mentioned: 

**Ore excavation** https://minecraft.curseforge.com/projects/ore-excavation
Give people an option to focus more on mod progression and less on resource gathering especially at the beginning.

**Crafting Tweaks** https://mods.curse.com/mc-mods/minecraft/233071-crafting-tweaks
Very handy when you need to make multiples of a complex recipe, especially balance grid and fill grid. 

## New suggestions:

**Hunger In Peace** https://minecraft.curseforge.com/projects/hunger-in-peace?gameCategorySlug=mc-mods&projectID=224473
So that we still have a use for all the HarvestCraft goodies while playing Peaceful survival.

**Chunk-Pregenerator** https://mods.curse.com/mc-mods/minecraft/267193-chunkpregenerator
Very thorough pregen tool with adjustable parameters, especially tick speed during generation. Good for overcoming block lag which I run into often.

**Comment by OrdinatorStouff** [comment]
@rosareven yeah you can parrot existing suggestions if you want to show your support for it as well (and even better [but not necessary] to contribute opinion/argument for why it should be included)

**Comment by rosareven** [comment]
@OrdinatorStouff Thanks. I polished up my suggestion post. Also removed Super Crafting Frame because I realised it's not updated to 1.11.2.

**Comment by rosareven** [comment]
@Atskadan Blbliocraft has clipboard with same functionality and similar recipe (just uses a bit more paper). Also paginated and can be hung on walls.

**Comment by OrdinatorStouff** [comment]
[Integrated Dynamics](https://minecraft.curseforge.com/projects/integrated-dynamics?gameCategorySlug=mc-mods&projectID=236307) (and maybe [Integrated Tunnels](https://minecraft.curseforge.com/projects/integrated-tunnels?gameCategorySlug=mc-mods&projectID=251389) )-- a sort of xnet-like mods but worth considering for more item/fluid/rf transfer options

**Comment by ghost** [comment]
> Industrial Foregoing is a reinvention of the old 1.7.10 Minefactory Reloaded. It will try to mimic the 1.7.10 version and enhance it with new machines and new mechanics.

https://minecraft.curseforge.com/projects/industrial-foregoing

**Comment by Ricket** [comment]
[MalisisDoors](https://minecraft.curseforge.com/projects/malisisdoors) -- animates doors, makes them more convenient, adds more types and custom door types.

**Comment by ghost** [comment]
Mob Grinding Utils - a mod that encourages players to create new ways for farming mobs for drops and experience.

Solar Flux Reborn - Another way to get RF power in the mod pack

torch Master - Adds a special torch which prevents mob spawning in a configurable radius.

Streams - This mod introduces real flowing rivers, with a true current, to Minecraft

Integrated Dynamics & Integrated Tunnels

Advanced Rocketry - about exploration and exploitation of other worlds and planets.

**Comment by Ricket** [comment]
Mob grinding utils, yes please!

Streams makes me nervous. All the releases are marked beta status. And IIRC, if you remove it from the modpack then your world is totally messed up from all the special stream water blocks going missing.

Advanced Rocketry looks like they're still hammering out a good number of bugs, maybe it would be good to let it cook a bit longer... (on the flip side, inclusion in ATM means lots more people to help find bugs, so there's that...)

**Comment by djcosmic2** [comment]
Mob Grinding Utils - Because there is no other way to auto kill mobs effectively beside the spike.

Silent's Gems - An old favorite when I played on Tlovetech's Love Clan Modpack. Adds gems that you can use to create weapons, armor, as well as methods of teleportation (with XP) and enchantment tokens (to add enchants to weapons)

Realistic Processing - a mod created by a fellow LoveClan Member, created for ModOff April 2017. It adds an early game "Bloomery" to ore double. "The Bloomery is an early game way to double your ores, it has to be manually attended to or it will go out and you will loose your progress on the smelt."

Woot - Lagless mob farms powered by RF

Water Strainer - Early Game misc resource gathering

Placeable Items - Place items on the ground as 3D items

Duck Craft - adds ducks.

Controlling - adds a search bar to the key bindings menu

Tumbleweed - to add some extra life to the game 

Random Things - Nice utility mod

Chest Transporter - For moving chests

**Comment by djcosmic2** [comment]
also Global XP -- I miss the enderIO liquid XP method of storing XP in a block

**Comment by JackyyTV** [comment]
Shameless advertising *hides*
Simple Sponge - https://minecraft.curseforge.com/projects/simple-sponge

**Comment by Ricket** [comment]
[Calculator](https://minecraft.curseforge.com/projects/calculator) -- have not played it but it looks awesome.

**Comment by ghost** [comment]
Draconic Evolution & Brandon's Core - If they fix the issues with it crashing sponge servers.(Reason a lot of mod packs removed it from 1.11 till there was a fix)

Not Enough Items 1.8 or Nei - For people who don't like using just JEI

Tiny Progressions - Helps with people who like to make farms.

MalisisDoors - More fancy doors for players bases 

NuclearCraft - Play with it in another mod packs and seems like a good fit for this one.
